src/main/java/com/rackspace/ceres/app/web/ParamUtils.java

```diff
@@ -44,4 +45,13 @@ static boolean paramPresentOrTrue(MultiValueMap<String, String> params, String k
       }
     }
   }
+
+  static String getTenant(String tenant, String tenantId) {
```

Comment:
Following from the previous content, please name the arguments more clearly like the following. Also the method name is probably better named avoiding "getter" terminology.
```suggestion
  static String resolveTenant(String tenantParam, String tenantHeader) {
```

and then it will be more obvious which ones hash higher precedence.